README.md

```diff
@@ -94,6 +94,38 @@ Run it like so:
 ./mkreleaselog v0.25.0 v0.26.0 > /tmp/v0.26.0.notes.txt
 ```
 
+## Contribution 
+
+### Maintainers
+
+The core maintainers of this repository are:
+- @nemo, @vmx & @shawnrader, from the fil-crypto team
```

Comment:
I'm not really a maintainer. I surely help out, but I think it should be @nemo and @shawnrader only.